moveit_setup_assistant_plugins/author_information_widget/package.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
```

Comment:
don't make this a separate package! we already have too many moveit packages. you can have all the plugins in the same moveit_setup_assistant package

Reply:
Okay I see, I'll just add the plugins to the msa package and not create any new one  :+1: 
I'll add a `plugins` directory  in the `moveit_setup_assistant`, that is okay?

Reply:
imho, I would prefer the plugins to be in a separate package, so the plugins would be more standalone which would make developing and adding new plugins easier and faster.

for example, if the `plugins` are part of the msa package, we would use relative paths to some headers in the package, but if we put the plugins in a separate package we wont do that, hence users wont have to add the plugin to the msa package to work, the plugins can then be exported from any package just by depending on the msa package for the headers instead of having to add to it.

Reply:
Just because the default plugins are defined within the MSA package, this doesn't imply that all plugins need to be added there. Also, you don't have to (and actually shouldn't) use relative include paths.

Reply:
Okay if the its known that the users wont have to add their plugins in the msa package, should be okay adding the plugins under the msa. I also think I wont have to use relative paths. Thanks.

Reply:
Yes I've did that.